docs/app/js/codepen.js

```diff
@@ -0,0 +1,175 @@
+(function() {
+  DocsApp
+    .factory('$codepenDataAdapter', CodepenDataAdapter)
```

Comment:
These services shouldn't start with a `$`; only services provided by Angular itself are supposed to.